Doc/whatsnew/3.11.rst

```diff
@@ -1155,6 +1155,10 @@ C API Changes
   be used for ``size``.
   (Contributed by Kumar Aditya in :issue:`46608`.)
 
+* :c:func:`_PyFrameEvalFunction` now takes :c:struct:`_PyInterpreterFrame`
+  as its second parameter, instead of :c:struct:`PyFrameObject`.
+  See PEP 523 for more details of how to use this function pointer type.
```

Comment:
```suggestion
* :c:func:`_PyFrameEvalFunction` now takes ``_PyInterpreterFrame*``
  as its second parameter, instead of ``PyFrameObject*``.
  See :pep:`523` for more details of how to use this function pointer type.
```